packages/ui/src/components/Designer/Canvas/index.tsx

```diff
@@ -80,9 +80,10 @@ interface Props {
   changeSchemas: ChangeSchemas;
   removeSchemas: (ids: string[]) => void;
   paperRefs: MutableRefObject<HTMLDivElement[]>;
+  sidebarOpen: boolean;
 }
 
-const Main = (props: Props, ref: Ref<HTMLDivElement>) => {
+const Canvas = (props: Props, ref: Ref<HTMLDivElement>) => {
```

Comment:
I renamed this to `Canvas` as it felt more specific than `Main`, but I will revert if you disagree.